packages/service-clients/end-to-end-tests/odsp-client/src/test/audience.spec.ts

```diff
@@ -86,8 +78,10 @@ describe("Fluid audience", () => {
 	 *
 	 * Expected behavior: upon resolving container, the partner member should be able
 	 * to resolve original member.
+	 *
+	 * Note: This test is currently skipped because the web app examples indicate the audience is functioning properly.
```

Comment:
Would be nice to include the work item number here (and other skipped tests) in case this ends up taking longer than expected to fix